examples/favorites_advanced/favorites_advanced_base/lib/src/extensions/puppy_extensions.dart

```diff
@@ -20,9 +20,9 @@ extension PuppUtils on Puppy {
         isFavorite: isFavorite ?? this.isFavorite,
         gender: gender ?? this.gender,
         breedType: breedType ?? this.breedType,
-        displayName: displayName ?? this.displayName,
+        displayName: displayName ?? this.displayName ,
```

Comment:
@StanislavPrime, have you formatted this by mistake ?

Reply:
Yes, I was trying to set null to displayName and to displayCharacteristics and after that I changed them back to the original code and I have not formatted the code after that. I formatted the code now.